docs/pages/blog/turbo-1-6-0.mdx

```diff
@@ -0,0 +1,209 @@
+---
+title: Turborepo 1.6
+date: 2022/10/19
+description: Turborepo 1.6 lets you use Turborepo in non-monorepos, prune for npm, and improves cache performance.
+tag: "web development"
+ogImage: /images/blog/turbo-1-6-0/twitter-card.png
+---
+
+import { Tabs, Tab } from '../../components/Tabs'
+
+# Turborepo 1.6
+
+import { Authors } from '../../components/Authors'
+import Callout from '../../components/Callout'
+
+<div className="text-sm mt-2 text-center text-gray-500 dark:text-gray-400">Wednesday, October 19th, 2022</div>
+
+<Authors authors={[ 'mattpocock', 'gsoltis', 'nathanhammond', 'tknickman', 'anthonyshewdev', 'jaredpalmer', 'mehulkar' ]} />
+
+Turborepo 1.6 changes the game for Turborepo - you can now use it in **any project**.
+
+- [**Turborepo in non-monorepos**](#any-codebase-can-use-turborepo): Seeing slow builds on your project? You can now use Turborepo to speed up builds in any codebase with a `package.json`.
+- [**`turbo prune` now supports npm**](#prune-now-supported-on-npm): Pruning your monorepo is now supported in monorepos using `npm`, completing support for all major workspace managers.
+- [**Faster caching**](#performance-improvements-in-the-cache): We've improved the way we handle local file writes, meaning a big speed-up of Turborepo's cache.
+
+Update today by running `npm install turbo@latest`.
+
+## Any codebase can use Turborepo
+
+Turborepo helps speed up tasks in your codebase. Until now, we'd built Turborepo specifically for monorepos - codebases which contain multiple applications and packages.
+
+Turborepo is fantastic in monorepos because they have so many tasks to handle. Each package and app needs to be built, linted, and tested.
+
+But we got to thinking: lots of codebases that _aren't_ monorepos run plenty of tasks. Most CI/CD processes do a lot of duplicated work that would benefit from a [cache](/docs/core-concepts/caching).
+
+So we're excited to announce that **any codebase can now use Turborepo**. Try it out now:
+
+### Add Turborepo to your project
+
+1. **Install `turbo` globally:**
+
+<Tabs items={['npm', 'yarn', 'pnpm']} storageKey="selected-pkg-manager">
+  <Tab>
+    ```bash
+    npm install --global turbo
```

Comment:
`--global` is deprecated and npm tells you to use `--location=global` now. Probably not something to change quite yet